src/app/containers/CpsAssetPageMain/index.jsx

```diff
@@ -13,6 +13,8 @@ import MediaPlayer from '../CpsAssetMediaPlayer';
 import Blocks from '../Blocks';
 import ATIAnalytics from '../ATIAnalytics';
 import cpsAssetPagePropTypes from '../../models/propTypes/cpsAssetPage';
+import fauxHeadline from '../FauxHeadline';
```

Comment:
something like `AriaHiddenHeadline` might be a better name

Reply:
`AriaHiddenHeadline` is a feature of this component, but I would say it is the main feature. The main feature is that this is visually shown the same as other `Headline`'s in Simorgh but is not a `<h1>`

Reply:
Ah cool - I missed the fact you were also passing `as=strong` to the psammead headline